README.md

```diff
@@ -220,6 +221,45 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
     msvcp140d_oss.dll
 ```
 
+# How To Run The Tests
+
+In order to run any of the tests you must be in a correctly configured environment for the architecture you are
+targeting, have python 3.8 or newer and have `clang-cl` on the PATH. If you are testing the x64 flavor of the STL then

```

Comment:
We shouldn't assume that every user knows what `clang-cl` is or where it comes from.
```suggestion
targeting, have python 3.8 or newer and have LLVM's `bin` directory on the PATH. If you are testing the x64 flavor of the STL then
```